<?xml version="2.0"?>

<launch>

	<node name="robot_movement_generation" type="baseline" pkg="human_robot_collaborative_learning" output="screen">
		<rosparam file="$(find human_robot_collaborative_learning)/config/baseline.yaml" command="load"/>
	</node>
	
</launch>
